cmd/addvalidators.go

```diff
@@ -147,10 +147,14 @@ func runAddValidatorsSolo(_ context.Context, conf addValidatorsConfig) (err erro
 		return errors.Wrap(err, "transform cluster manifest")
 	}
 
-	// TODO(xenowits): Write cluster backup, see https://github.com/ObolNetwork/charon/issues/2345.
+	// Save cluster backup to disk
+	//nolint:revive,staticcheck
+	if loadMetadata.IsLegacyLock {
+		// TODO(xenowits): Write cluster backup, see https://github.com/ObolNetwork/charon/issues/2345.
```

Comment:
#2345 says otherwise?

> Note that backup file should only be created when the cluster is loaded from a cluster manifest file, not a legacy lock file.